client/components/section-header/test/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
```

Comment:
I usually leave out the `External dependencies` comment if there are none.

Reply:
6:1  error  Missing external, internal dependencies docblocks  wpcalypso/import-docblock

Reply:
Added external dependencies so it doesn't matter either way.